examples/gno.land/p/demo/uint256/uint256_test.gno

```diff
@@ -0,0 +1 @@
+package uint256
```

Comment:
We can drop this file completely, since there seem to be unit tests in other separate `_test.gno` files